api/cronTasks/scheduler.js

```diff
@@ -1,14 +1,17 @@
 const cron = require('node-cron');
```

Comment:
So what this is doing on schedule determining who the winning team is and then assigning the points to the child? So the total points won in the matchup only go to a child of the winning team? Or the points go to the child who's submission was voted best?